Tests/LoggerTests/LogEntryDecoderTests.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  LogEntryDecoderTests.swift
+//  
+//
+//  Created by Radek Čep on 22.02.2022.
+//
+
+import XCTest
+@testable import Logger
+
+class LogEntryDecoderTests: XCTestCase {
+    func test_sut_should_parse_line() throws {
+        let dateFormatter = DateFormatter.monthsDaysTimeFormatter
+        let expectedDate = dateFormatter.date(from: "02-21 15:16:17.189")
+        let record = "|> [INFO 02-21 15:16:17.189] FileName.swift - Function - line 42: Some log with special characters ::[]{}()//"
+
+        let sut = LogEntryDecoder(
+            logRecordSeparator: "|>",
```

Comment:
Testy encoderu/decodery jsou napsané pro jine separátory než výchozí konfigurace, která bude pravděpodobně nejpoužívanější. Rozšířil bych testy o nejpoužívanější Use Case, nebo změnil výchozí konfigurace na to co je v testech.